libtracepoint-decode-cpp/include/tracepoint/PerfEventInfo.h

```diff
@@ -30,36 +36,67 @@ namespace tracepoint_decode
 
     struct PerfSampleEventInfo
     {
-        uint64_t id;                            // Always valid if GetSampleEventInfo succeeded.
```

Comment:
Note: Breaking changes to this struct. Added an `event_desc` pointer, and several fields are now stored there instead of directly in this struct.